### ISSUE TITLE
RP conda package recipe - fix dependencies

### ISSUE THREAD

**Issue by mtitov** [radical-cybertools/radical.pilot#2471]
List of dependencies in conda-forge should be updated ([current list](https://github.com/conda-forge/radical.pilot-feedstock/blob/master/recipe/meta.yaml#L22-L31))
Should be
```yaml
    - python
    - radical.utils >=1.8.4
    - radical.saga >=1.8.0
    - pymongo < 4
    - setproctitle
```

**Comment by mtitov** [comment]
fixed in PR [#89](https://github.com/conda-forge/radical.pilot-feedstock/pull/89) - ticket will be closed after merging changes

**Comment by mtitov** [comment]
resolved!
